src/PackageWatcher.ts

```diff
@@ -76,8 +77,25 @@ export class PackageWatcher {
      * This will update the context keys and trigger a `resolve` task,
      * which will in turn update the Package Dependencies view.
      */
-    async handlePackageChange() {
+     async handlePackageSwiftChange() {
+        // Load SwiftPM Package.swift description 
         await this.ctx.swiftPackage.reload();
+        // Create launch.json files based on package description. Run this in parallel
+        // with package resolution
+        debug.makeDebugConfigurations(this.ctx);
+        // if package has dependencies resolve them
+        if (this.ctx.swiftPackage.dependencies.length > 0) {
+            await commands.resolveDependencies();
+        }
+    }
+
+    /**
+     * Handles a create or change event for **Package.swift** and **Package.resolved**.
```

Comment:
This documentation (and the one for `handlePackageSwiftChange`) is now a bit out-of-date because we have separate methods for both files.

Reply:
ok will fix